openmdao/solvers/nonlinear/newton.py

```diff
@@ -85,6 +85,8 @@ def _declare_options(self):
                              desc='Set to True to turn on sub-solvers (Hybrid Newton).')
         self.options.declare('max_sub_solves', type_=int, default=10,
                              desc='Maximum number of subsystem solves.')
+        self.options.declare('initial_sub_solve', type_=bool, default=False,
```

Comment:
If I understand this correctly, this option means that we only subsolve the very first time this solver runs, and then thereafter, we don't perform any subsolves?